tests/test_anomaly_detectors.py

```diff
@@ -17,9 +18,11 @@
     "learner_constructor,auc,cli_string",
     [
         (partial(HalfSpaceTrees, window_size=100, number_of_trees=25, max_depth=15), 0.54, None),
+        (partial(OnlineIsolationForest, num_trees=32, max_leaf_samples=32, window_size=2048), 0.52, None),
```

Comment:
This window size is maybe too large the dataset ``ElectricityTiny`` is only 1k long.